NetSDK/Services/Client/Classes/SelfRefreshingClient.cs

```diff
@@ -90,8 +92,9 @@ private void ReadConfig(ConfigurationOptions config)
             HttpReadTimeout = config.ReadTimeout ?? 15000;
             SdkVersion = "net-" + Version.SplitSdkVersion;
             SdkSpecVersion = "net-" + Version.SplitSpecVersion;
-            SdkMachineName = config.SdkMachineName;
-            SdkMachineIP = config.SdkMachineIP;
+            SdkMachineName = Environment.MachineName;
+            var ipAdresses = Dns.GetHostAddresses(Environment.MachineName);
+            SdkMachineIP = ipAdresses.Last().ToString();
```

Comment:
ok, I can keep it in config, and if config values are null I set it in this automated way.